db/migrate/20221104183606_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.string :password_digest
+      t.string :phone
+      t.string :address
+      t.string :role
```

Comment:
Kindly for this project, let's use the boolean for the role, by saying t.string :admin, default: false then we will be updating the role manually using the console